static/locales/de.json

```diff
@@ -543,61 +568,87 @@
   "Girafarig": "Girafarig",
   "Giratina": "Giratina",
   "Glaceon": "Glaziola",
+  "Glacial Lure": "Gletscher-Lockmodul",
+  "Glacial Lures": "Gletscher-Lockmodul",
   "Glaciate": "Eiszeit",
   "Glalie": "Firnontor",
   "Glameow": "Charmian",
   "Glare": "Giftblick",
+  "Glastrier": "Polaross",
   "Gligar": "Skorgla",
   "Gliscor": "Skorgro",
   "Gloom": "Duflor",
   "Gogoat": "Chevrumm",
   "Golbat": "Golbat",
   "Goldeen": "Goldini",
+  "Golden Razz Berry": "Goldene Himmihbeere(n)",
   "Golduck": "Entoron",
   "Golem": "Geowaz",
   "Golett": "Golbit",
+  "Golisopod": "Tectass",
   "Golurk": "Golgantes",
   "Goodra": "Viscogon",
   "Goomy": "Viscora",
   "Gorebyss": "Saganabyss",
+  "Gossifleur": "Cottini",
   "Gothita": "Mollimorba",
   "Gothitelle": "Morbitesse",
   "Gothorita": "Hypnomorba",
   "Gourgeist": "Pumpdjinn",
   "Granbull": "Granbull",
+  "Grapploct": "Kaocto",
   "Grass Knot": "Strauchler",
   "Grass Pledge": "Pflanzensäulen",
   "Grass Whistle": "Grasflöte",
   "Grass": "Pflanze",
   "Grassy Terrain": "Grasfeld",
   "Graveler": "Georok",
   "Gravity": "Erdanziehung",
+  "Great Ball": "Superball",
+  "Greedent": "Schlaraffel",
   "Greninja": "Quajutsu",
   "Grimer": "Sleima",
+  "Grimmsnarl": "Olangaar",
+  "Grookey": "Chimpep",
   "Grotle": "Chelcarain",
   "Groudon": "Groudon",
   "Ground": "Boden",
   "Grovyle": "Reptain",
   "Growl": "Heuler",
   "Growlithe": "Fukano",
   "Growth": "Wachstum",
+  "Grubbin": "Mabula",
   "Grudge": "Nachspiel",
   "Grumpig": "Groink",
+  "Grunt": "Rüpel",
   "Guard Split": "Schutzteiler",
   "Guard Swap": "Schutztausch",
   "Guillotine": "Guillotine",
   "Gulpin": "Schluppuck",
+  "Gumshoos": "Manguspektor",
   "Gunk Shot": "Mülltreffer",
   "Gurdurr": "Strepoli",
   "Gust": "Windstoß",
+  "Guzzlord": "Schlingking",
   "Gyarados": "Garados",
+  "Gym cells": "Arena Zellen",
+  "Gym leader": "Arenaboss",
+  "Gym levels": "Arenalevel",
+  "Gym sidebar": "Arena-Seitenleiste",
+  "Gyms & Raids": "Arenen & Raids",
+  "Gyms in battle only": "Nur Arenen mit Raidkampf",
```

Comment:
This is not related to raids. It related to a gym is under attack by another player!

Reply:
okay is changed